frontend/lib/laletterbuilder/letter-builder/my-letters.tsx

```diff
@@ -22,12 +26,145 @@ export const LaLetterBuilderMyLetters: React.FC<ProgressStepProps> = (
   );
 };
 
+interface CompletedLetterCardProps {
+  pdfBytes: string;
+}
+
+const CompletedLetterCard: React.FC<CompletedLetterCardProps> = (props) => {
+  const { children, pdfBytes } = props;
+
+  const downloadPdf = () => {
+    const bytes = atob(pdfBytes);
+    const byteChars = bytes.split("").map((el, i) => bytes.charCodeAt(i));
+    const pdfBlob = new Blob([new Uint8Array(byteChars)], {
+      type: "application/pdf;base64",
+    });
+    const pdfUrl = URL.createObjectURL(pdfBlob);
+    window.open(pdfUrl);
+  };
+
+  return (
+    <div className="jf-la-letter-card">
+      <div className="content">
+        <StaticImage
+          ratio="is-32x32"
+          src={getLaLetterBuilderImageSrc("repair-tools")}
+          alt=""
+        />
+        <h2>
+          <Trans>Notice to repair letter</Trans>
+        </h2>
+        {children}
+        {pdfBytes && (
+          <button
+            className="button is-primary is-medium jf-is-next-button"
+            onClick={downloadPdf}
+          >
+            <Trans>Download letter</Trans>
+          </button>
+        )}
+      </div>
+      <hr />
+      <Accordion question={"What's next?"} questionClassName="">
+        <h2>
+          <Trans>Allow 14 days for a response</Trans>
+        </h2>
+        <p>
+          <Trans>
+            If your landlord or property manager doesn’t respond, you should
+            file a complaint.
+          </Trans>
+        </p>
+        <span>
+          <Trans>For LA City residents</Trans>
+        </span>
+        <p>
+          <Trans>Call LAHD at (866) 557-7368</Trans>
+        </p>
+
+        <span>
+          <Trans>For LA county residents</Trans>
+        </span>
+        <p>
+          <Trans>Call LADBS at (213) 473-3231</Trans>
+        </p>
+        <h2>
+          <Trans>Mark your calendar</Trans>
+        </h2>
+        <p>
+          <Trans>
+            Make sure you keep your schedule clear during the requested access
+            dates:
+          </Trans>
+        </p>
+      </Accordion>
+    </div>
+  );
+};
+
 const MyLettersContent: React.FC = (props) => {
+  const { session } = useContext(AppContext);
+  const processedLetters = session.habitabilityLetters?.filter(
+    (el) => !!el.fullyProcessedAt && !el.letterSentAt
```

Comment:
I generally prefer descriptive variable names to `el`, just helps you figure out what type it is. Maybe `letter`?